### PR TITLE
ci: Prepare for the 1.1.0 release

### PR DESCRIPTION
**Pull request by edbamolvyavahare**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -55,6 +55,6 @@ release:
     - glob: 'terraform-registry-manifest.json'
       name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
   # If you want to manually examine the release before its live, uncomment this line:
-  draft: false
```
Comment (wai-wong-edb) [inline .goreleaser.yml:58]: I think we need draft: false rather than commenting it out

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## v1.1.0 (October 14. 2024)
```
Comment (wai-wong-edb) [inline CHANGELOG.md:1]: Flesh out features/enhancements/bug fixes

File: docs/resources/faraway_replica.md [hunk 3]
```diff
@@ -228,8 +252,17 @@ Read-Only:
...
 
-- `id` (String) Cluster architecture ID. For example, "single" or "ha".For Extreme High Availability clusters, please use the [biganimal_pgd](https://registry.terraform.io/providers/EnterpriseDB/biganimal/latest/docs/resources/pgd) resource.
 - `name` (String) Name.
-- `nodes` (Number) Node count.
+
+## Import
+
+Import is supported using the following syntax:
+
+{{codefile "shell" "/Users/amol.vyavahare/Documents/projects/terraform-provider-biganimal/examples/resources/biganimal_faraway_replica/import.sh"}}
```
Comment (wai-wong-edb) [inline docs/resources/faraway_replica.md:268]: we need to change https://github.com/EnterpriseDB/terraform-provider-biganimal/blob/a60af1e2ae12c52325c00d196c3a25b49dcf7dac/templates/resources/faraway_replica.md.tmpl#L25 to `{{ codefile "shell" .ImportFile }}` and run make docs again

**Review by wai-wong-edb — CHANGES_REQUESTED** [review]
Some change requests. README.md needs to change from 1.0.0 to 1.1.0 too

File: .goreleaser.yml (hunk 1 above)
Comment (edbamolvyavahare) [reply .goreleaser.yml:58]: Done

File: docs/resources/faraway_replica.md (hunk 3 above)
Comment (edbamolvyavahare) [reply docs/resources/faraway_replica.md:268]: Done

**Review by wai-wong-edb — APPROVED** [review]
LGTM

**Pull request merged**